rpa-home/modules/ROOT/pages/invoke-rpa-from-flow-project.adoc

```diff
@@ -0,0 +1,18 @@
+
+# Create an Automation Project
+
+Create an automation project first. An automation project consists of the process, its requirements, the team that is responsible for the automation, and the implementation. 
```

Comment:
I suggest putting the first sentence as a bullet in the Before You Begin section and adding the details in sub-bullets or text under the first bullet. Do you need the details about the automation project here or does the reference take care of this?

Reply:
There are four steps to include RPA into Salesforce Flow:

To run RPA processes as part of a Salesforce Flow, follow these steps:

. xref:invoke-rpa-from-flow-project.adoc[Create an automation project.]
. xref:invoke-rpa-from-flow-build.adoc[Build your process in RPA Builder.]
. xref:invoke-rpa-from-flow-publish.adoc[Publish an invocable run configuration to Salesforce in RPA Manager.]
. xref:invoke-rpa-from-flow-run.adoc[Call the Invocable Actions from Salesforce Flow.]

This topic is the first step.

I think the explanation of a project is helpful for the (Salesforce/first time) user reading this.

I put in the H2s for consistency with our other topics and for easier overview

Reply:
I agree that there is not much content in each of the paragraphs